charts/civil-ccd/values.enableHmc.preview.template.yaml

```diff
@@ -0,0 +1,52 @@
+civil-ccd-stack:
+  enabled: false
+
+civil-service:
+  java:
+    environment:
+      CORE_CASE_DATA_API_URL: http://ccd-data-store-api-aat.service.core-compute-aat.internal
+      HMC_API_URL: http://hmc-cft-hearing-service-aat.service.core-compute-aat.internal
+      HMC_HEARINGS_SUBSCRIPTION_ENABLED: true
+      HMC_HEARINGS_TOPIC_NAME: hmc-to-cft-aat
+      HMC_HEARINGS_TOPIC_NAMESPACE: hmc-servicebus-aat
+      HMC_HEARINGS_TOPIC_POLICY_NAME: SendAndListenSharedAccessKey
+      HMC_HEARINGS_TOPIC_SUBSCRIPTION_NAME: "hmc-to-civil-subscription-pr-${CHANGE_ID}"
+      HMC_HEARINGS_TOPIC_THREAD_COUNT: 4
+
+civil-general-applications:
+  java:
+    environment:
+      CORE_CASE_DATA_API_URL: http://ccd-data-store-api-aat.service.core-compute-aat.internal
+
+xui-webapp:
+  enabled: true
+  nodejs:
+    environment:
+      SERVICES_HEARINGS_COMPONENT_API: http://jurisdiction-hearings-api-aat.service.core-compute-aat.internal
+      SERVICES_HEARINGS_COMPONENT_API_SSCS: http://${SERVICE_NAME}-sscs-hearings-api
+      HEALTH_CCD_COMPONENT_API: https://gateway-ccd.aat.platform.hmcts.net/health
+      HEALTH_CCD_DATA_API: http://ccd-data-store-api-aat.service.core-compute-aat.internal/health
+      SERVICES_CCD_COMPONENT_API: https://gateway-ccd.aat.platform.hmcts.net
+      SERVICES_EM_ANNO_API: https://gateway-ccd.aat.platform.hmcts.net
+      SERVICES_CCD_DATA_STORE_API: http://ccd-data-store-api-aat.service.core-compute-aat.internal
+      SERVICES_HMC_HEARINGS_COMPONENT_API: http://hmc-cft-hearing-service-aat.service.core-compute-aat.internal
+      SERVICES_HEARINGS_COMPONENT_API_CIVIL: https://civil-ccd-pr-${CHANGE_ID}.preview.platform.hmcts.net
+      SERVICES_HEARINGS_CASETYPES_CIVIL: CIVIL-${CHANGE_ID}
+      HEARINGS_JURISDICTIONS: CIVIL
+      LAUNCH_DARKLY_CLIENT_ID: 5de6610b23ce5408280f2268
```

Comment:
should this be here and not in key vault or somewhere?

Reply:
it is just an id, not a password, it is a protected lower environment and is basically just a copy from SSCSs